ethcontract/src/transaction/kms.rs

```diff
@@ -162,8 +162,8 @@ impl Account {
         encoder.append(&v);
         // RLP encoding doesn't allow leading zeros for s & r, yet default H256 RLP encoding preserves leading 0s
         // By converting and encoding U256, we get rid of the leading zeros.
-        encoder.append(&U256::from(signature.r.as_bytes()));
-        encoder.append(&U256::from(signature.s.as_bytes()));
+        encoder.append(&U256::from_big_endian(signature.r.as_bytes()));
```

Comment:
unfortunately H256 doesn't have as_big_endian (to make it clear that this is indeed compatible). Have to rely on the docs https://rust.velas.com/primitive_types/struct.H256.html#note

Reply:
Since `H256` is just a `[u8; 32]`, I don't think a concept of endianness makes sense... It is 32 bytes, so they are in byte order.